packages/agents/lighthouse/src/tasks/prover/operations/consumer.ts

```diff
@@ -41,7 +41,7 @@ export const consume = async () => {
           channel.ack(message);
         } catch (err: unknown) {
           logger.error("Processing messaages failed", requestContext, methodContext, undefined, { err });
-          channel.reject(message);
```

Comment:
I am just curious why `channel.reject(message, false)` has been removed before.

Reply:
It was added.
Would it not be a better design to drop the message if execute failed ?
It would get retried again, later right ?

Reply:
Its a better design to drop the message if execute failed. If we do nothing in a failure case, it will reenqueued and the backlog get bigger.  It has been originally added here when I am doing refactors with amqplib so I was asking someone tried to remove it but why???  lol